tracing-subscriber/src/fmt/format/json.rs

```diff
@@ -23,7 +23,15 @@ use tracing_log::NormalizeEvent;
 ///
 /// The full format includes fields from all entered spans.
 #[derive(Default, Debug, Copy, Clone, Eq, PartialEq)]
-pub struct Json;
+pub struct Json {
+    pub(crate) flatten_event: bool,
+}
+
+impl Json {
+    pub(crate) fn flatten_event(&mut self, flatten_event: bool) {
```

Comment:
I think it's probably safe to make this a public API, but we don't need to in this PR...